sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md

```diff
@@ -74,25 +74,25 @@ custom-types: CloudEvent,EventGridEvent
 model-override-setter-from-superclass: true
 
 input-file:
```

Comment:
Could you just reference the event grid readme?

Reply:
Actually, I meant why do we need to list each service separately? In .NET we reference the root readme.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Azure.Messaging.EventGrid/src/autorest.md

Reply:
Ah. I see what you meant now. Do you mean using the eventgrid rest spec readme file, so it will pick up the latest SHA automatically without us modifying the SHA every time?

Reply:
In .NET at least, we still need to modify the SHA every time, but the benefit is that you are referencing a single file rather than dozens of files.

Reply:
Created an issue to be addressed in the next release along with other autogen configurations. https://github.com/Azure/azure-sdk-for-java/issues/26882